.github/scripts/check_diff.py

```diff
@@ -55,8 +59,9 @@ def add_dependents(dirs_to_eval: Set[str], dependents: dict) -> List[str]:
 
     if len(files) == 300:
```

Comment:
```suggestion
    if len(files) >= 300:
```

Reply:
fwiw this still goes through the regular addition of things to test as well after this check. so if they ever bump limit to > 300 it would actually work correctly and at exactly 300 would be overly eager to CI things